azurerm_storage_account/vars.tf

```diff
@@ -39,6 +39,12 @@ variable "access_tier" {
   type = string
 }
 
+variable "blob_properties_delete_retention_policy_days" {
+  type = number
+  description = "Enable soft delete policy and specify the number of days that the blob should be retained"
+  default = null
```

Comment:
You missed to align values and the attribute; I did this mistake thousand of time. It's not blocking.
Just install di terraform plugin in visual studio code or run terraform fmt in the module directory.

Reply:
thank you, fixed